libraries/src/Plugin/CMSPlugin.php

```diff
@@ -75,6 +76,14 @@ abstract class CMSPlugin implements DispatcherAwareInterface, PluginInterface
 	 */
 	protected $allowLegacyListeners = true;
 
+	/**
+	 * The application object
+	 *
+	 * @var    CMSApplicationInterface
+	 * @since  __DEPLOY_VERSION__
```

Comment:
```suggestion
	 *
	 * @since  __DEPLOY_VERSION__
```

i think there is a supposed to be an empty line before since